test/extended/apiserver/health_endpoints.go

```diff
@@ -115,3 +164,15 @@ func testPath(transport http.RoundTripper, path string, requiredChecks sets.Set[
 	}
 	return nil
 }
+
+func checkFlatform(ctx context.Context, name string, oc *exutil.CLI) error {
+	isMicroShift, err := exutil.IsMicroShiftCluster(oc.AdminKubeClient())
+	framework.ExpectNoError(err)
+	if isMicroShift {
+		return fmt.Errorf("MicroShift does not have this component %s", name)
```

Comment:
and here i would return `false, "MicroShift does not have this component %s", name"